src/OrchardCore.Modules/OrchardCore.Users/Services/DefaultControllerService.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Logging;
+using OrchardCore.DisplayManagement;
+using OrchardCore.Email;
+using OrchardCore.Entities;
+using OrchardCore.Modules;
+using OrchardCore.Settings;
+using OrchardCore.Users.Events;
+using OrchardCore.Users.Models;
+using OrchardCore.Users.ViewModels;
+
+namespace OrchardCore.Users.Services
+{
+    public class DefaultControllerService
```

Comment:
UserService?

Reply:
I think we already have `IUserService`, so I don't want confuse the people ;)